_posts/2020-06-09-solidity-summit-recap.md

```diff
@@ -103,6 +103,6 @@ We hope to see or hear many of you soon again, either in Gitter, Github issues,
 In the meantime, please feel free to reach out to us with your ideas on how to improve the interaction between the Solidity team and the community at any time. 
 
 ---
-Follow us on [Twitter](https://twitter.com/solidity_lang)<br>
-Ask us anything on [Gitter](https://gitter.im/ethereum/solidity)<br>
-Check out Solidity on [GitHub](https://github.com/ethereum/solidity)<br>
+Follow us on [Twitter](https://twitter.com/solidity_lang)
```

Comment:
I think this won't work since it puts all the three items into one line without the `<br>`, I've had this live without `<br>` and added them later since it all appeared in one line without them.